apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/RefineModal/steps/2_SelectEvaluationResults/SelectableEvaluationResultsTable.tsx

```diff
@@ -124,7 +134,11 @@ export const SelectableEvaluationResultsTable = ({
               )}
             >
               <TableCell>
-                <input type='checkbox' checked={isSelected} />
+                <input
+                  type='checkbox'
+                  checked={isSelected}
+                  onChange={toggleSelection(evaluationResult)}
```

Comment:
@csansoon React was complaining because a checkbox with prop `checked` has to have `onChange`